apps/files_sharing/src/views/SharingDetailsTab.vue

```diff
@@ -154,7 +154,7 @@
 						:disabled="!canSetDownload"
 						:checked.sync="canDownload"
 						data-cy-files-sharing-share-permissions-checkbox="download">
-						{{ t('files_sharing', 'Allow download') }}
+						{{ t('files_sharing', 'Allow Download/Sync') }}
```

Comment:
Also the casing is wrong, either:
```suggestion
						{{ t('files_sharing', 'Allow download / sync') }}
```

or

```suggestion
						{{ t('files_sharing', 'Allow download and sync') }}
```

Reply:
updated as suggested.